.eslintrc

```diff
@@ -1,12 +1,14 @@
 {
   "parser": "babel-eslint",
+  "rulePaths": ["build-system/eslint-rules/"],
+  "plugins": ["eslint-plugin-google-camelcase"],
   "ecmaFeatures": {
     "modules": true,
     "arrowFunctions": true,
     "blockBindings": true,
-    "forOf": false,
-    "destructuring": false,
-    "spread": false
+    "forOf": true,
```

Comment:
@jridgewell i don't believe so (not 100% sure, ill check), but it won't be transpiled in any way. ill go ahead and add a presubmit check for that too if we already don't have one